src/ec/packages/ec-component-ecl-compliance/components/ecl-compliance-footer.html.twig

```diff
@@ -0,0 +1,145 @@
+{#
+  Parameters:
+    identity: (object),
+    sections: (array of objects),
+    common: (object)
+#}
+
+{% extends "@ecl-twig/ec-component-ecl-compliance/ecl-compliance.html.twig" %}
+
+{% set component = 'footer' %}
+{% set message = '' %}
+{% set prefix = '' %}
+{% set section_number = '' %}
+{% set no_identity_title_case = false %}
+{% set no_identity_follow_case = false %}
+{% set no_identity_follow_links_case = false %}
+{% set no_identity_follow_links_link_case = false %}
+{% set no_identity_follow_label_case = false %}
+{% set no_identity_info_link_case = false %}
+{% set no_section_links_case = false %}
+{% set no_section_link_case = false %}
+{% set no_common_link_keys_case = false %}
+{% set failing_sections = [] %}
+
+{% set no_identity_case = identity is empty %}
+{% if no_identity_case == false %}
+  {% set no_identity_title_case = identity.title is not defined or identity.title is empty %}
+  {% set no_identity_follow_case = identity.follow is not defined or identity.follow is empty %}
+  {% if no_identity_follow_case == false %}
+    {% set no_identity_follow_label_case = identity.follow.label is not defined or identity.follow.label is empty %}
+    {% set no_identity_follow_links_case =
+      identity.follow.links is not defined or
+      identity.follow.links is empty or
+      identity.follow.links is not iterable %}
+  {% endif %}
+  {% if no_identity_follow_links_case == false %}
+    {% for item in identity.follow.links %}
+      {% if (item.link is not defined or item.link is empty) or
+        (item.link.label is not defined or item.link.label is empty) or
+        (item.link.path is not defined or item.link.path is empty) %}
+        {% set no_identity_follow_links_link_case = true %}
+      {% endif %}
+    {% endfor %}
+  {% endif %}
+
+  {% set no_identity_info_case = identity.info is not defined or identity.info is empty or identity.info is not iterable %}
```

Comment:
The list of checks here is long and is not readable, could be made as this one https://github.com/ec-europa/ecl-twig/pull/466/files#diff-9cccea82e3b9f8a5b77f4707ea690d91R31